src/models/index.js

```diff
@@ -16,7 +17,11 @@ export default {
   phs,
   humidex,
   net,
+  clo_tout,
+  clo_tout_array,
   wbgt,
+  clo_tout,
+  clo_tout_array,
```

Comment:
adding these functions twice?

Reply:
Ah the merge was _very_ ugly, I'll fix that